File: 2413.最小偶倍数.js
/*
 * @lc app=leetcode.cn id=2413 lang=javascript
 *
 * [2413] 最小偶倍数
 */

// @lc code=start
/**
 * @param {number} n
 * @return {number}
 */
var smallestEvenMultiple = function(n) {
    return n % 2 == 0 ? n : n * 2;
};
// @lc code=end

File: 2413.最小偶倍数.java
/*
 * @lc app=leetcode.cn id=2413 lang=java
 *
 * [2413] 最小偶倍数
 */

// @lc code=start
class Solution {
    public int smallestEvenMultiple(int n) {
        return n % 2 == 0 ? n : 2 * n;
    }
}
// @lc code=end

